dask/tests/test_distributed.py

```diff
@@ -546,6 +546,39 @@ def f(d, a):
     await c.compute(df.map_partitions(f, arr, meta=df._meta))
 
 
+def test_map_partitions_df_input():
+    """
+    Check that map_partitions can handle a delayed
+    partition of a dataframe input
+    """
+    pd = pytest.importorskip("pandas")
+    dd = pytest.importorskip("dask.dataframe")
+    dist = pytest.importorskip("distributed")
```

Comment:
We can drop this line as we skip the entire module if `distributed` isn't installed

https://github.com/dask/dask/blob/d2ce17c73918dff45981eb8d9c64f7bbe48fc595/dask/tests/test_distributed.py#L3

Reply:
Fixed